.github/scripts/end2end/patch-coredns.sh

```diff
@@ -14,6 +14,7 @@ corefile="
     rewrite name exact ci-zenko-aws-crr-target-bucket.aws-mock.zenko.local ingress-nginx-controller.ingress-nginx.svc.cluster.local
     rewrite name exact ci-zenko-aws-fail-target-bucket.aws-mock.zenko.local ingress-nginx-controller.ingress-nginx.svc.cluster.local
     rewrite name exact ci-zenko-aws-target-bucket.aws-mock.zenko.local ingress-nginx-controller.ingress-nginx.svc.cluster.local
+    rewrite name exact ci-zenko-aws-replication-ctst-bucket.aws-mock.zenko.local ingress-nginx-controller.ingress-nginx.svc.cluster.local
```

Comment:
why do we need a new endpoint?
can't we use the existing locations (and/or mocks) for the tests?

(this would simplify setup quite a bit, by reducing setup and avoiding the introduction of yet more variables ; plus avoid creating an extra location and associated delay...)

Note that it may be better to use a local/mock location instead of AWS, to avoid "integration" issues that may happen if AWS is down and ensure we fully control (in the tests) all buckets. I think we already have such locations: $RING_S3C_BACKEND_SOURCE_LOCATION and $RING_S3C_BACKEND_SOURCE_NON_VERSIONED_LOCATION

Reply:
- it is kind of expensive in terms of setup (hoping that some day we want to run the tests locally and/or against any artesca, the more location there are the more it may be complicated), and passing all the parameters to the tests
- also zenko _today_ does not scale well with the number of locations. There is no (enforced) hard limit, but we document in artesca that we should not have more than 10 locations. (There are multiple reasons to this: the list is added to some k8s CR; in places the code traverses the whole list of locations; many locations create many kafka consumers in some backbeat processes, which can overload the node workers; ....)